woocommerce/rest-api/Controllers/Settings.php

```diff
@@ -125,7 +125,35 @@ public function get_items_permissions_check( $request ) {
 	// TODO: get_items()
 
 
-	// TODO: get_item()
+	/**
+	 * Gets a single setting.
+	 *
+	 * @since x.y.z
+	 *
+	 * @param \WP_REST_Request $request request object
+	 * @return \WP_REST_Response|\WP_Error
+	 */
+	public function get_item( $request ) {
+
+		$setting_id = $request->get_param( 'id' );
+
+		if ( $setting = $this->settings->get_setting( $setting_id ) ) {
+
+			return $this->prepare_item_for_response( $setting, $request );
```

Comment:
No need to call `rest_ensure_response` because `prepare_item_for_response` already calls it.